### PR TITLE
Update us_pbs.m3u

### PR DESCRIPTION
**Pull request by what-a**
Unfortunately, most PBS streams are now offline. This pull request will label those no longer working as [Offline]. I'm guessing the bot could not detect them as offline since they were originally geo-blocked.

### CODE REVIEW

**Comment by Central302** [comment]
@what-a PBS Thirteen https://wnetdt.lls.pbs.org/out/v1/0456457548354b88b32fc437e4e7ee01/index.m3u8 [USA IP ONLY]
Finding WTTW Chicago

File: channels/us_pbs.m3u [hunk 1]
```diff
@@ -1,277 +1,277 @@
...
 https://wpbsdt.lls.pbs.org/out/v1/e3680a91029c4df9b7797ce13d828207/index.m3u8
-#EXTINF:-1 tvg-id="WXELTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wxel-color-cobranded-logo-q5DODDu.png" group-title="Local",PBS West Palm Beach FL (WXEL) (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="WXELTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wxel-color-cobranded-logo-q5DODDu.png" group-title="Local",PBS West Palm Beach FL (WXEL) (1080p) [Offline]
 https://wxeldt.lls.pbs.org/out/v1/d4f2bc8357164a2e93d35abd2caecc4b/index.m3u8
-#EXTINF:-1 tvg-id="KPTSTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/kpts-color-cobranded-logo-THeGpUD.png" group-title="Local",PBS Wichita KS (KPTS) (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="KPTSTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/kpts-color-cobranded-logo-THeGpUD.png" group-title="Local",PBS Wichita KS (KPTS) (1080p) [Offline]
 https://kptsdt.lls.pbs.org/out/v1/1ca3404d6bc3404c9daa86c8f1ae19d0/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/us_pbs.m3u:277]: https://kptsdt.lls.pbs.org/out/v1/dc566b3d244f42a1b17654b8d983d9db/index.m3u8 ?

File: channels/us_pbs.m3u [hunk 2]
```diff
@@ -1,277 +1,277 @@
...
+#EXTINF:-1 tvg-id="WFSUTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wfsu-color-cobranded-logo-ok3uUgp.png" group-title="Local",PBS Tallahassee FL (WFSU) (1080p) [Offline]
 https://wfsudt.lls.pbs.org/out/v1/d2be172139764358a0d54352c8411845/index.m3u8
-#EXTINF:-1 tvg-id="WEDUTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="Local",PBS Tampa FL (WEDU) (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="WEDUTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="Local",PBS Tampa FL (WEDU) (1080p) [Offline]
 https://wedudt.lls.pbs.org/out/v1/3e147f13bc464958b6ace4e5a5b9accc/index.m3u8
 #EXTINF:-1 tvg-id="WGTETV.us" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMjYxNjkxMTBf/PBS.png" group-title="Local",PBS Toledo OH (WGTE) (1080p) [Geo-blocked]
 https://pbs.lls.cdn.pbs.org/est/index.m3u8?WVPB=
```
Comment (LaneSh4d0w) [inline channels/us_pbs.m3u:257]:
```suggestion
https://pbs.lls.cdn.pbs.org/est/index.m3u8
```

Not useful it seems to keep the query parameter.

**Comment by Central302** [comment]
PBS National Feed : East https://pbs.lls.cdn.pbs.org/est/index.m3u8
West : https://pbs.lls.cdn.pbs.org/pst/index.m3u8

**Comment by what-a** [comment]
Central302 helped me realize that we had some channels leeching off of the national feed. Those are now eliminated and the east and west feeds are added. That brings us to 5 working streams:
image

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**